docs/controls/Loading.md

```diff
@@ -0,0 +1,60 @@
+# Loading XAML Control 
+
+The loading control is for showing an animation with some content when the user should wait in some tasks of the app.
+
+## Syntax
+
+An example of how we can build the loading control.
+
+``` xaml
+<controls:Loading x:Name="LoadingControl" HorizontalAlignment="{Binding HorizontalAlignment.Value}" 
```

Comment:
Let's remove the alignment, background and opacity bindings for the documentation. Add in a binding for IsLoading